EMML/1001-2000/EMML1832.xml

```diff
@@ -45,35 +146,930 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                   </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="ink">Black, red.</seg>
+                        <seg type="script"/>
+                        <seg type="rubrication">Incipits, numbers.</seg>
+                        <date notBefore="1250" notAfter="1300">Script of the thirteenth century.</date>
+                     </handNote>     
+                  </handDesc>
                   <decoDesc>
-                     <decoNote xml:id="d1">
-                        <desc>Decorative frames, ff. 7ab, 8ab, 9ab, 10ab, 11ab and 12ab. A cross in an illustrious frame, f. 12b</desc>
+                     <decoNote xml:id="d1" type="frame">
+                        <locus target="#5v"/>
+                        <desc>Decorative frame depicting <persName ref="PRS5633IyasusM"/>.</desc>
+                     </decoNote>
+                     <decoNote xml:id="d2" type="frame">
+                        <locus from="7r" to="8r"/>
+                        <desc>Finely illuminated frames surrounding the text of <ref type="work" corresp="LIT1349EpistlEusebius"/>.</desc>
+                     </decoNote>
+                     <decoNote xml:id="d3" type="frame">
+                        <locus from="8v" to="12v"/>
+                        <desc>Decorative frames surrounding <ref type="work" corresp="LIT1224Canons"/>.</desc>
+                     </decoNote>
+                     <decoNote xml:id="d4" type="miniature">
+                        <locus target="#12r"/>
+                        <desc>Finely executed full-page <ref type="authFile" corresp="AT1034Tempietto"/> depicting a pavillon with a number of 
+                           birds including two peacocks accompanied by two gazelles representing the Fountain of Life.</desc>
+                     </decoNote>
+                     <decoNote xml:id="d5"  type="miniature">
+                        <locus target="#12v"/>
+                        <desc>A cross in an illustrious frame.</desc>
+                     </decoNote>
+                     <decoNote xml:id="d6" type="miniature">
+                        <locus target="#13r"/>
+                        <desc>Miniature depicting <persName ref="PRS6751Mark"/> in a circular frame, which in turn is placed in a rectangular 
+                           frame.</desc>
+                        <q xml:lang="gez">ርእሰ፡ ማርቆስ። </q>
+                     </decoNote>
+                     <decoNote xml:id="d7" type="miniature">
+                        <locus target="#13v"/>
+                        <desc>Miniature depicting <persName ref="PRS7743Paul"/>.</desc>
+                        <q xml:lang="gez">ጳውሎስ። </q>
+                     </decoNote>
+                     <decoNote xml:id="d8" type="miniature">
+                        <locus target="#14r"/>
+                        <desc>Miniature depicting <persName ref="PRS0000000000">St. Timothy</persName>.</desc>
```

Comment:
ID missing?

Reply:
According to my investigation, an ID for St. Timothy does not exist yet. It was decided to postpone the creation of a new ID in order to avoid unhappy identifications. Do you think, I should delete the mark-up?

Reply:
either mark up and provide ID or do not mark up
do not leave placeholder refs
**not marking up is perfectly fine and this is what I have been asking - to not mark up**